client/features/home/Home.jsx

```diff
@@ -10,12 +11,36 @@ class Home extends React.Component {
     super(props);
 
     this.state = {
-      suggested: data.suggested,
-      history: data.history,
-      trending: data.trending
+      suggested: [],
+      history: [],
+      trending: []
     };
   }
 
+  componentDidMount() {
+    //if the prev props id is different from current and if id is not null
+    //when this page is hit - it should run a ajax request to server 
+    console.log(this.props.currentId)
+    //for the current id 
+
+
+    if (this.props.currentId !== null && this.props.currentId) {
```

Comment:
`if (this.props.currentId)` would have the same effect

Reply:
good point - had no time to refactor, this was all slapped out in a night